solidity/test/utils/ILlamaPay.sol

```diff
@@ -13,4 +13,5 @@ interface ILlamaPay {
   function withdraw(address _from, address _to, uint216 _amountPerSec) external;
   function modifyStream(address _oldTo, uint216 _oldAmountPerSec, address _to, uint216 _amountPerSec) external;
   function pauseStream(address _to, uint216 _amountPerSec) external;
+  function claim(address _llamaVestAddress) external;
```

Comment:
this interface is inaccurate.

Reply:
yes.

Reply:
@dristpunk in the console, try to run `cast interface [CONTRACT_ADDRES]` and paste the result in the file.

Reply:
we dont need this interface anymore, deleted